src/Reflection/BetterReflection/SourceLocator/ComposerJsonAndInstalledJsonSourceLocatorMaker.php

```diff
@@ -112,18 +107,12 @@ public function create(string $projectInstallationPath): ?SourceLocator
 		);
 
 		foreach ($classMapDirectories as $classMapDirectory) {
-			if (!is_dir($classMapDirectory)) {
```

Comment:
we don't need this call (and also not the one below for `is_dir`), because of

```
$classMapFiles = array_filter($classMapPaths, 'is_file');
$classMapDirectories = array_filter($classMapPaths, 'is_dir');
```

above